deploy/Chart/templates/controller/deployment.yaml

```diff
@@ -27,6 +27,15 @@ spec:
       {{- end }}
     spec:
       serviceAccountName: controller
+      initContainers:
+      - name: bicep
+        image: "{{ .Values.bicep.image }}:{{ .Values.bicep.tag | default $appversion }}"
+        command: ['sh', '-c', 'mv /bicepconfig.json /bicepconfig/bicepconfig.json && mv /bicep /usr/local/bin/bicep']
```

Comment:
Do we need to consider handling a user-provided `bicepconfig.json`. i.e. one that is used within the user's Git repo

Reply:
users can provide a bicepconfig.json in their git repo and bicep build should use that one